### DISCUSSION TITLE
Fehlermeldung bei Upload KNX-PROJEKT

### DISCUSSION THREAD

**Discussion by jurkoe** [brutella/hkknx-public#184] [General]
Hallo, beim Upload des KNX-Projektes kommt die Fehlermeldung „Multipart: Nextpart: Bufio: bufferfull„. Bis dahin hat es immer wunderbar funktioniert. Danke für Euer Feedback.

LG Eneriko

**Comment by brutella** [comment]
Wieviel freien Speicherplatz hast du auf dem Computer?

**Reply by jurkoe** [reply]
Korrekt, Partition war zu klein.
